Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -283,6 +302,92 @@ namespace AzToolsFramework::Prefab
         return GetInstanceReference(m_rootAliasFocusPath);
     }
 
+    LinkId PrefabFocusHandler::AppendPathFromFocusedInstanceToPatchPaths(PrefabDom& providedPatch, const AZ::EntityId& entityId) const
+    {
+        LinkId linkId = InvalidLinkId;
+
+        if (!providedPatch.IsArray())
+        {
+            AZ_Error("Prefab", false, "Patch is not an array of updates. Update failed.");
+            return linkId;
+        }
+
+        // Grab the owning instance.
+        InstanceOptionalReference owningInstance = m_instanceEntityMapperInterface->FindOwningInstance(entityId);
+        AZ_Assert(owningInstance != AZStd::nullopt, "Owning Instance is null");
+
+        // Retrieve the path from the focused prefab instance to the owningInstance of the entityId.
+        AZStd::string prefix;
+        InstanceOptionalReference focusedPrefabInstance = GetInstanceReference(m_rootAliasFocusPath);
+
+        {
+            // Climb up the instance hierarchy from this instance until you hit the focused prefab instance.
+            InstanceOptionalConstReference instance = owningInstance;
+            AZStd::vector<InstanceOptionalConstReference> instancePath;
+
+            while (&instance->get() != &focusedPrefabInstance->get() && instance.has_value())
+            {
+                instancePath.emplace_back(instance);
+                instance = instance->get().GetParentInstance();
+            }
+
+            if (&instance->get() != &focusedPrefabInstance->get())
+            {
+                AZ_Error(
+                    "Prefab", false,
+                    "AppendPathFromFocusedInstanceToPatchPaths - entityId is not owned by a descendant of the focused prefab instance.");
+                return linkId;
+            }
+
+            for (auto instanceIter = ++instancePath.rbegin(); instanceIter != instancePath.rend(); ++instanceIter)
+            {
+                prefix.append("/Instances/");
+                prefix.append((*instanceIter)->get().GetInstanceAlias());
+            }
```

Comment:
You can use the recently added PrefabInstanceUtils::ClimbUpToTargetInstance for this.